docs/source/exposing.md

```diff
@@ -0,0 +1,267 @@
+kind: ScyllaCluster
+spec:
+  exposeOptions:
+    broadcastOptions:
+       clients:
+         type: ServiceLoadBalancerIngress
+         podIP:
+           source: Status
+```
+
+## Deployment Examples
+
+In this section, we will showcase specific examples of various network scenarios and explain how nodes and clients communicate with one another.
+### In-cluster only
+
+ScyllaCluster definition:
+```yaml
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+spec:
+  exposeOptions:
+    nodeService:
+      type: ClusterIP
+    broadcastOptions:
+      clients:
+        type: ServiceClusterIP
+      nodes:
+        type: ServiceClusterIP
+```
+
+Both client and nodes are deployed within the same Kubernetes cluster. 
+They talk through IP addresses from ClusterIP Service.
+Because ClusterIP Services are only routable within the same Kubernetes cluster, connection from outside is impossible.
+
+![ClusterIPs](static/exposing/clusterip.svg)
+
+### In-cluster node-to-node, VPC clients-to-nodes
+
+ScyllaCluster definition:
+```yaml
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+spec:
+  exposeOptions:
+    nodeService:
+      type: ClusterIP
+    broadcastOptions:
+      clients:
+        type: PodIP
+      nodes:
+        type: ServiceClusterIP
+```
+
+In this scenario, we assume that the Pod IP subnet is routable within a VPC. 
+Clients within the VPC network can communicate directly with ScyllaCluster nodes using Pod IPs.
+Nodes communicate with each other exclusively within the same Kubernetes cluster.
+
+![PodIPs](static/exposing/podips.svg)
+
+### Multi DC
+
+ScyllaCluster definition:
+```yaml
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+spec:
+  exposeOptions:
+    nodeService:
+      type: Headless
+    broadcastOptions:
+      clients:
+        type: PodIP
+      nodes:
+        type: PodIP
+```
+
+In this scenario, we set up two separate Kubernetes clusters in different regions, each within its respective VPC. 
+These VPCs are interconnected to facilitate inter-VPC connectivity. 
+We operate on the assumption that the Pod IP subnet is routable within each VPC. 
+Nodes, deployed in these distinct regions, broadcast their Pod IP addresses, enabling them to establish connections with one another. 
+Clients, whether deployed within the same Kubernetes cluster or within a VPC, have the capability to reach nodes using their Pod IPs.
+
+Since there is no requirement for any address other than the Pod IP, the 'Headless' service type is sufficient.
+
+
+![MultiDC](static/exposing/multidc.svg)
+
+### Internet
+
+ScyllaCluster definition:
+```yaml
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+spec:
+  exposeOptions:
+    nodeService:
+      type: LoadBalancer
+    broadcastOptions:
+      clients:
+        type: ServiceLoadBalancerIngress
+      nodes:
+        type: ClusterIP 
+```
+
+We assume that a Kubernetes cluster has been deployed in a cloud provider environment that supports external load balancers. 
```

Comment:
since you've already shown how to configure it generically, I wonder if these should pick a particular platform like EKS or GKE and show it really e2e, even if only for one.

Reply:
although, maybe we can defer that